views/albums.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="albums-container">
+    {{#each }}
+    <img src="{{}}" alt="">
```

Comment:
Here, you could use the same structure as the image link in the artist-search-results.hbs file.